src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -101,22 +94,23 @@
     </Style>
 
     <Style x:Key="AccentBrowseButtonStyle" TargetType="Button" BasedOn="{StaticResource AccentButtonStyle}">
-        <Setter Property="FontSize" Value="{StaticResource StandardFontSize}"/>
         <Setter Property="ToolTipService.Placement" Value="Mouse"/>
         <Setter Property="Margin" Value="10,0,0,0"/>
         <Setter Property="VerticalAlignment" Value="Bottom"/>
     </Style>
 
+    <Style x:Key="IconButtonTextBlockStyle" TargetType="TextBlock">
+        <Setter Property="Margin" Value="10,0,0,0"/>
+    </Style>
+
     <!--Slider-Related Styling-->
     <Style x:Key="SliderSettingStyle" TargetType="Slider" BasedOn="{StaticResource DefaultSliderStyle}">
-        <Setter Property="FontSize" Value="{StaticResource StandardFontSize}"/>
         <Setter Property="ToolTipService.Placement" Value="Mouse"/>
     </Style>
 
     <Style x:Key="SliderValueLabelStyle" TargetType="TextBlock">
         <Setter Property="Width" Value="35"/>
         <Setter Property="Margin" Value="10,0,0,20"/>
-        <Setter Property="FontSize" Value="15"/>
         <Setter Property="HorizontalAlignment" Value="Left"/>
         <Setter Property="VerticalAlignment" Value="Center"/>
         <Setter Property="ToolTipService.Placement" Value="Mouse"/>

```

Comment:
Check the `CustomSliderControlGridStyle`'s width below. 300 might look too big now, especially with all of the font sizes being way smaller.